.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Automatically handle line endings
+* text=auto
+
+# Force line endings for these files
+*.js  text eol=lf
+*.cjs text eol=lf
+*.mjs text eol=lf
+
+# Don't add these files to git archives
+CHANGELOG.md export-ignore
```

Comment:
I would keep the `CHANGELOG.md` file. It's a small file with very useful information.

Reply:
No problem. I'll update the `.gitattributes`.